src/main/java/stsjorbsmod/cards/cull/Apparate.java

```diff
@@ -0,0 +1,56 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.VulnerablePower;
+import com.megacrit.cardcrawl.vfx.combat.CleaveEffect;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Apparate extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Apparate.class);
+
+    private static final CardRarity RARITY = CardRarity.BASIC;
+    private static final CardTarget TARGET = CardTarget.ALL;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int DAMAGE = 10;
+    private static final int SELF_VULNERABLE = 3;
+    private static final int ENEMY_VULNERABLE = 3;
+    private static final int UPGRADE_PLUS_DAMAGE = 3;
+
+    public Apparate() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = DAMAGE;
+        magicNumber = baseMagicNumber = ENEMY_VULNERABLE;
+        urMagicNumber = baseUrMagicNumber = SELF_VULNERABLE;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        addToBot(new ApplyPowerAction(p, p, new VulnerablePower(p, urMagicNumber, false)));
+        addToBot(new VFXAction(new CleaveEffect()));
+        addToBot(new DamageAllEnemiesAction(p, DamageInfo.createDamageMatrix(damage), DamageInfo.DamageType.NORMAL, AbstractGameAction.AttackEffect.NONE));
```

Comment:
`DamageInfo.createDamageMatrix` is the suggested way for powers to calculate multi-damage, but for cards it's instead suggested to set `isMultiDamage = true;` in the constructor and use the built-in `multiDamage` field as the second argument to the damage all action; it results in more correct interaction with certain combinations of vulnerable/etc on only some of the enemies. See `Fireball` for an example.